helm/prometheus-rules/templates/loki-ruler-datasource-configmap.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.mimir.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: loki-ruler-datasource-{{ .Values.managementCluster.name }}
+  namespace: monitoring
+  labels:
+    app.giantswarm.io/kind: datasource
+data:
+  loki-ruler-{{ .Values.managementCluster.name }}.yaml: |
+    apiVersion: 1
+    datasources:
+    - access: proxy
+      editable: false
+      jsonData:
+        manageAlerts: true
+        httpHeaderName1: X-Scope-OrgID
+      secureJsonData:
+        httpHeaderValue1: {{ .Values.managementCluster.name }}
```

Comment:
Any reason why we add a tenant info here? Mimir multi-tenancy is not enabled IIRC

Reply:
Oh that's in loki my bad